pkgs/applications/misc/warpd/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libXi
+, libXinerama
+, libXft
+, libXfixes
+, libXtst
+, libX11
+, libXext
+
+, waylandSupport ? false, cairo, libxkbcommon, wayland
+}:
+
+stdenv.mkDerivation rec {
+  pname = "warpd";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "rvaiya";
+    repo = "warpd";
+    rev = "e618c26b721919a97b0fa9da89501fda271ae53d";
+    sha256 = "yfqv+aABPSpw62BXpKXtBOp4CdmZfGSks+PojjsM/Vw=";
```

Comment:
You would also need to add `leaveDotGit = true` so that you can pass a Make flag for [this](https://github.com/rvaiya/warpd/blob/c3878e3bf69c97aa0f79f7bdf71d13cafbe1600b/src/warpd.c#L324) as its done [here](https://github.com/rvaiya/warpd/blob/c3878e3bf69c97aa0f79f7bdf71d13cafbe1600b/Makefile#L1).